Sources/xcproj/PBXGroup.swift

```diff
@@ -106,11 +106,11 @@ extension PBXGroup: PlistSerializable {
     }
 
     fileprivate func name(reference: String, proj: PBXProj) -> String? {
-        if let group: PBXGroup = proj.getCachedReference(reference) {
+        if let group: PBXGroup = proj.objects.getReference(reference) {
```

Comment:
These can just use the normal `objects.groups`. It will be much more efficient

Reply:
Good catch! I was updating references to get everything building but didn't reevaluate callers of the general getReference func